### PR TITLE
LINK-1921 & LINK-1922 | Talpa Accounts Management

### PR DESCRIPTION
**Pull request by harriris-vincit**
### Description
Adds the possibility to add and edit Talpa accounts in Django admin and using the API. Deleting is not possible as it is not possible in Talpa either. However, it is possible to make an account inactive in Linked Events to disable it.

Note: unlike in the similar [Talpa merchants PR](https://github.com/City-of-Helsinki/linkedevents/pull/871), the API calls for accounts management will be done in a separate Jira issue and PR. This is because the so-called product mapping needs to be done in Talpa first before accounts can be created for product mappings. Also, product mapping and accounting has been planned to be done when creating a registration for an event instead of doing it in the admin "phase" / views.
### Closes
[LINK-1921](https://helsinkisolutionoffice.atlassian.net/browse/LINK-1921)
[LINK-1922](https://helsinkisolutionoffice.atlassian.net/browse/LINK-1922)

[LINK-1921]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-1921?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[LINK-1922]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-1922?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
**LINKEDEVENTS-API branch is deployed to platta: https://linkedevents-pr874.api.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**LINKEDEVENTS-API branch is deployed to platta: https://linkedevents-pr874.api.dev.hel.ninja** :rocket::rocket::rocket:

**Review by nicobav — APPROVED** [review]
:shipit:

**Comment by terovirtanen** [comment]
**LINKEDEVENTS-API branch is deployed to platta: https://linkedevents-pr874.api.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**LINKEDEVENTS-API branch is deployed to platta: https://linkedevents-pr874.api.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**LINKEDEVENTS-API branch is deployed to platta: https://linkedevents-pr874.api.dev.hel.ninja** :rocket::rocket::rocket:

File: events/permissions.py [hunk 1]
```diff
@@ -180,7 +181,10 @@ def has_object_permission(self, request: Request, view, obj):
         if (
             settings.WEB_STORE_INTEGRATION_ENABLED
             and request.method in ("PUT", "PATCH")
-            and request.data.get("web_store_merchants")
+            and (
+                request.data.get("web_store_merchants")
+                or request.data.get("web_store_accounts")
+            )
```
Comment (jorilindell) [inline events/permissions.py:187]: Not really related to this PR, but user who has both admin and financial admin permissions should be able to update all fields of the organization

**Review by jorilindell — APPROVED** [review]
Looks very good and works as expected.

Added one comment which is more like a reminder to allow admin + financial admin user to update organization

**Pull request merged**